source/_components/media_player.pioneer.markdown

```diff
@@ -23,10 +23,12 @@ To add a Pioneer receiver to your installation, add the following to your `confi
 media_player:
   - platform: pioneer
     host: 192.168.0.10
+    port: 23
```

Comment:
`port` is optional. Our sample should only show the minimal needed part and port defaults to 23 according your PR.